tests/test-recipes/metadata/_overlinking_detection/bld.bat

```diff
@@ -0,0 +1,4 @@
+@echo on
+echo int main() { return 0; }>main.c
+cl -c -EHsc -GR -Zc:forScope -Zc:wchar_t -Fomain.obj main.c
+link -out:%PREFIX%\Library\bin\main.exe main.obj
```

Comment:
I have no idea how these got created/committed, removing shortly

Reply:
Removed in https://github.com/conda/conda-build/pull/4621/commits/4859edcfcfd6a7eacd5627907fa8b1cab1c5503f

Reply:
conda-build's test suite must build "in place" in its own test data, where it will write this kind of script. First edit .gitignore. In January, copy test data to temporary directory before running test suite.

Reply:
yeah that's what I'm thinking